src/Run/CommandLine.cs

```diff
@@ -0,0 +1,1876 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.Execute;
+
+using System.Reflection;
+
+namespace Microsoft.Fx.CommandLine
```

Comment:
Don't worry about pruning for now but I would like to file a follow-up issue to merge your changes back into the shared copy, as that is the better option. Even if you are using new specific stuff we should be able to add that as an option to the shared copy so we can keep things consistent and shared.